### DISCUSSION TITLE
buddhism/samu/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#11] [General]
# buddhism/samu/

https://shanenull.com/buddhism/samu/

**Comment by shane0** [comment]
Find source for quote

Let go or be dragged

**Comment by shane0** [comment]
use the record for this 
https://shanenull.com/buddhism/record/

**Comment by shane0** [comment]
https://forms.gle/1AyVd9ZDFXMrdFeL9

**Comment by shane0** [comment]
THE SEVEN POINTS OF POSTURE
Attend to each in turn when you first take your meditation seat.
If you become uncomfortable while sitting, you may go through these points again.
1. Legs
2. Arms
3. Back
4. Eyes
5. Jaw
6. Tongue
7. Head and shoulders

Sit cross legs or stand
